pkg/netpol/eval/resources.go

```diff
@@ -22,10 +23,11 @@ type (
 	// PolicyEngine encapsulates the current "world view" (e.g., workloads, policies)
 	// and allows querying it for allowed or denied connections.
 	PolicyEngine struct {
-		namspacesMap map[string]*k8s.Namespace                // map from ns name to ns object
-		podsMap      map[string]*k8s.Pod                      // map from pod name to pod object
-		netpolsMap   map[string]map[string]*k8s.NetworkPolicy // map from namespace to map from netpol name to its object
-		cache        *evalCache
+		namspacesMap      map[string]*k8s.Namespace                // map from ns name to ns object
+		podsMap           map[string]*k8s.Pod                      // map from pod name to pod object
+		netpolsMap        map[string]map[string]*k8s.NetworkPolicy // map from namespace to map from netpol name to its object
+		ownersToLabelsMap map[string]map[string]string             // map from ownerReference name to its labels map
```

Comment:
```suggestion
		ownersToLabelsMap map[string]map[string]string             // map from pods' ownerReference name to its labels map
```

Reply:
should be in the context of a namespace (there could be two owners of the same name in different namespaces).